contracts/staking/PoSValidatorManager.sol

```diff
@@ -35,6 +35,10 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         mapping(bytes32 delegationID => bytes) _pendingRegisterDelegatorMessages;
         /// @notice Maps the delegationID to pending end delegator messages.
         mapping(bytes32 delegationID => bytes) _pendingEndDelegatorMessages;
+        /// @notice Maps the delegationID to its pending staking rewards.
+        mapping(bytes32 delegationID => uint256) _pendingDelegatorRewards;
+        /// @notice Maps the validator owner address to its pending staking rewards.
+        mapping(address validatorOwner => uint256) _pendingValidatorRewards;
```

Comment:
I think this might be better named as `unclaimedXRewards`. "Pending" makes me think that this should calculate how many rewards a staker would get if they exited now. I think we could make this one map with the key being the address that can claim the rewards.

Reply:
i would go with `redeemableRewards`